spec/lib/baw-workers/harvest/single_file_spec.rb

```diff
@@ -183,10 +183,10 @@
       password = 'password'
       auth_token = 'auth token this is'
 
-      file_hash = 'SHA256::c6d561c91664a92a1598bda3b79734d5ee29266ad0411ffaea1188f29d5b6439'
+      file_hash = 'SHA256::b06458c96ac2de75d77b83e1b3acb3e64b84a4bfe671c794153855bf7b5e96fd'
```

Comment:
Related to the changes below as well: what information is stored in the database?

It appears the hash, original file name, and relative path all refer to the wav file.

- The hash maps to the `original_audio` file so it should be the hash of the file that is stored - the WAV file 👍 
- However, other two fields refer to the original source file. Up until #36, and #35 the source files have never been converted so their definition has been unambiguous. I'm leaning towards changing their meaning to mean "original_file_name before conversion". Thoughts?
  - it would be nice for the behaviour to be the same as the FLAC use case in #36 however, I think WAac is a throwaway format and the use case might be slightly different